packages/prepare/src/BasePrepare.ts

```diff
@@ -1,10 +1,27 @@
-import { Texture, BaseTexture } from '@pixi/core';
+import { Texture, BaseTexture, AbstractRenderer, Renderer } from '@pixi/core';
```

Comment:
Not sure if both interfaces should be used here.
Would be nice to leave it to `AbstractRenderer`

Reply:
also, maybe `import type` will be better here?